decidim-participatory_processes/app/models/decidim/participatory_process.rb

```diff
@@ -55,6 +58,12 @@ class ParticipatoryProcess < ApplicationRecord
     scope :upcoming, -> { where(arel_table[:start_date].gt(Time.current)) }
     scope :active, -> { where(arel_table[:start_date].lteq(Time.current).and(arel_table[:end_date].gt(Time.current).or(arel_table[:end_date].eq(nil)))) }
 
+    scope :visible_participatory_processes_for, lambda { |user|
```

Comment:
You can rename this to `visible_for`, since it's clear it will return processes, and we skip a few keystrokes